grammars/tex.cson

```diff
@@ -50,7 +50,7 @@
         'name': 'comment.line.percentage.semicolon.texshop.tex'
       }
       {
-        'begin': '^(%\\s*!T[eE]X)\\s+(\\w+)\\s*='
+        'begin': '^(%\\s*!T[eE]X)\\s+([^:]+?\\s*:\\s*)?(\\$?\\w+)\\s*='
```

Comment:
Basically LGTM, but I suppose the extended syntaxes are not common among other LaTeX IDEs such as TeXShop or TeXworks so it would be better if DiCy's magic comments have their own `name` (e.g., `comment.line.percentage.magic.dicy.tex`). What do you think?

Reply:
Sounds good!